x-pack/plugins/alerts/server/plugin.ts

```diff
@@ -252,14 +258,11 @@ export class AlertingPlugin {
     core: CoreSetup
   ): IContextProvider<RequestHandler<unknown, unknown, unknown>, 'alerting'> => {
     const { alertTypeRegistry, alertsClientFactory } = this;
-    return async (context, request) => {
+    return async function alertsRouteHandlerContext(context, request) {
```

Comment:
curious why you changed this to a named function from the unnamed lambda; debugging?

Reply:
I... honestly can't remember 😮 
I have no memory of doing this.

_Might_ have been because there was no more need for the `this` access once the call to `this.getScopedClientWithAlertSavedObjectType` was removed 🤔 
But I'm amused/worried by the fact that I can't remember changing it.

Reply:
haha, figured it out!
It's a merge conflict resolution.

I had two branches going in parallel and this PR was merged last week: https://github.com/elastic/kibana/pull/66719/files#diff-63b3de2748c01e5f4fd1927a82f9114bL250

When I then merged master into this branch they clashed - and I had to resolve it manually.
Since the user of `this.getScopedClientWithAlertSavedObjectType` was introduced in the other PR, and I coudl see that there was no need for the `this` in the new state of the code - I reverted back to what was there *before* that other PR was merged into master- the version with `function alertsRouteHandlerContext`.

that's why I have no memory of writing that code - I didn't.
I just let it revert back to that when fixing a merge conflict. 😆